src/interfaces/filing-interfaces/filing-interfaces.ts

```diff
@@ -185,7 +185,11 @@ export interface RestorationFilingIF {
     nameTranslations?: NameTranslationIF[]
     nameRequest?: RegistrationNameRequestIF
     parties: PartyIF[]
-    offices: IncorporationAddressIF
-    contactPoint: ContactPointIF
+    offices: RegisteredRecordsAddressesIF | object
```

Comment:
What are you trying to do here that requires a possibly untyped object?

Reply:
Frankly, just followed the example in Incorporation application case. I can take a look and fix this.

Reply:
Yeah, I know. That was a work-around for the "missing prop in typed object" issue -- kind of thing you've been bumping into and I mentioned.

If your code doesn't work with the object type then just leave it -- we will have a future ticket to fix such things.